govc/volume/snapshot/create.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright (c) 2023 VMware, Inc. All Rights Reserved.
```

Comment:
linter expects a range even for single year, `2023-2023`. I've an update to license.sh for this in a pending PR #3083 3083

Reply:
Fixed.